src/coreclr/vm/callhelpers.cpp

```diff
@@ -468,11 +468,32 @@ void MethodDescCallSite::CallTargetWorker(const ARG_SLOT *pArguments, ARG_SLOT *
 
                 switch (stackSize)
                 {
+#if defined(TARGET_LOONGARCH64)
+                    case 1:
+                        if (m_argIt.GetArgType() == ELEMENT_TYPE_U1)
+                            *((INT64*)pDest) = (UINT8)pArguments[arg];
+                        else
```

Comment:
Does this need to handle simple wrappers of primitive types (structs with just one field)?

They should be sign or zero extended based on the underlying type according to this comment: 
https://github.com/loongson/LoongArch-Documentation/pull/32#discussion_r772267496

Reply:
I think there is no need.
Becasue the field of struct will be pared again while using it.

Reply:
I think you are saying that the comment https://github.com/loongson/LoongArch-Documentation/pull/32#discussion_r772267496 is not valid.

The calling convention details like this have to be very well understood and described. Could you please work on clarifying the ABI for simple wrappers of primitive types with your colleagues?

Reply:
> I think you are saying that the comment [loongson/LoongArch-Documentation#32 (comment)](https://github.com/loongson/LoongArch-Documentation/pull/32#discussion_r772267496) is not valid.

I am sorry for descriped unclear.
At the comment, using a register passing the `struct {int a;}` is signed extend.

> 
> The calling convention details like this have to be very well understood and described. Could you please work on clarifying the ABI for simple wrappers of primitive types with your colleagues?

This PR is args passing by stack, the struct {...} which sizeof(struct) is less than or equal 4-byte, the upper bits are undefined.
But we signed extend the upper bits except the unsigned types, it will be ok and simply the cases.